packages/expo-font/src/memory.ts

```diff
@@ -1,2 +1,32 @@
-export const loaded: { [name: string]: boolean } = {};
+import ExpoFontLoader from './ExpoFontLoader';
 export const loadPromises: { [name: string]: Promise<void> } = {};
+
+// cache the value on the js side for fast access to the fonts that are loaded
+let _cache: { [name: string]: boolean } = {};
+
+export const markLoaded = (fontFamily: string) => {
```

Comment:
If there is no purpose (capturing `this`), please use regular functions. Syntax for regular functions looks cleaner + such functions are hoisted which sometimes helps in ordering them within a file.
There are only two primary purposes to use arrow functions over regular functions: capturing `this` and callbacks.

```suggestion
export function markLoaded(fontFamily: string) {
```

Reply:
renamed